winlogbeat/Makefile

```diff
@@ -12,7 +12,3 @@ gen:
 # This is called by the beats packer before building starts
 .PHONY: before-build
 before-build:
-	# Windows
-	sed -i 's|#\{0,1\}\(to_files:\).*|\1 true|' $(PREFIX)/$(BEATNAME)-win.yml
-	sed -i 's|#\{0,1\}\(level:\).*|\1 info|' $(PREFIX)/$(BEATNAME)-win.yml
-	sed -i '/log files/{n;s|#\{0,1\}path:.*|path: C:/ProgramData/winlogbeat/Logs|}' $(PREFIX)/$(BEATNAME)-win.yml
```

Comment:
@andrewkroh I removed these as the current defaults should work fine and I want to minimize the number of seds we do on the config files.